src/views/home/home.spec.jsx

```diff
@@ -44,6 +44,6 @@ describe('Home View component', () => {
   });
 
   it('should have a BlogPostsList component', () => {
-    expect(home.find(BlogPostsList).length).toEqual(1);
+    expect(home.find(NavigationBar).find(BlogPostsList)).toBeTruthy();
```

Comment:
is it something to do with the addition of `onMountEnter` that would cause this test to fail now?  I would have expected it to cause the DOM structure of the home page to change, but i could be wrong. 🤔

Reply:
@thescientist13 : If we try printing home.find(BlogPostsList).html(), it is going to be empty. So the test is not able to find any BlogPostsList that is mounted. I just discovered that `expect(home.find(NavigationBar).find(BlogPostsList)).toBeTruthy();` is also a wrong assertion because even  expect(home.find('#abc')).toBeTruthy(); is going to give back a ReactWrapper , that's a truthy object.

Reply:
Ahh, I see, because now we are conditionally loading both lists.

So I wonder if this means we should update our test?  Maybe by default there should be `0` `BlogPostList` but if we fire a mouse scroll event or something with enzyme, then it would appear?

@DevLab2425 
What are your thoughts on this?  Seems like a mix of what we want as default behavior, and then test for that.  Curious where you land on this one.

Reply:
@thescientist13 : Yes you are right, if you try .simulate('click') or something like this in the test, it can be asserted in the test